locale/es.yml

```diff
@@ -1248,113 +1248,113 @@ __TRAIT__: ""
 # |  LANGUAGE: PYTHON  |
 # ----------------------
 # << Keywords >>
-  False: ""
-  None: ""
-  True: ""
-  and: ""
-  as: ""
-  assert: ""
-  async: ""
-  await: ""
-  break: ""
-  class: ""
-  continue: ""
-  def: ""
-  del: ""
-  elif: ""
-  else: ""
-  except: ""
-  finally: ""
-  for: ""
-  from: ""
-  global:  ""
-  if: ""
-  import: ""
-  in: ""
-  is: ""
-  lambda: ""
-  nonlocal: ""
-  not: ""
-  or: ""
-  pass: ""
-  raise: ""
-  return: ""
-  try: ""
-  while: ""
-  with: ""
-  yield: ""
+  False: "Falso" # ✅✅✅✅✅✅✅
+  None: "Ninguno" # ✅✅✅✅✅✅✅
+  True: "Verdadero" # ✅✅✅✅✅✅✅
+  and: "Y" # ✅✅✅✅✅✅✅
+  as: "como" # ✅✅✅✅✅✅
+  assert: "afirmar" # ✅❓ What about "asegurar"? ✅ I agree with "afirmar" ✅ ✅ I also agree with "afirmar"✅✅ I agree with "afirmar"
+  async: "asíncrono" # ✅✅✅✅✅✅✅
+  await: "esperar" # ✅✅✅✅✅✅✅
+  break: "interrumpir" # ✅✅✅✅✅Also "romper"✅ Yes, it also could be "romper"
```

Comment:
I still think interrumpir should translate for break - romper's direct translation does mean break, but I've mainly used the word to mean rip or ripping something off in English, so this sounds wrong to me.